source/common/config/utility.cc

```diff
@@ -40,6 +40,8 @@ void Utility::translateApiConfigSource(const std::string& cluster, uint32_t refr
 
   api_config_source.mutable_refresh_delay()->CopyFrom(
       Protobuf::util::TimeUtil::MillisecondsToDuration(refresh_delay_ms));
+  api_config_source.mutable_request_timeout()->CopyFrom(
```

Comment:
Since v1 is hard coded effectively, can we set 1s here and hard code it? Or just leave empty and then when the proto is pulled from it will just default to 1s? So I'm thinking almost all these changes can be reverted.

Reply:
Addressed in 0ecfc41.

Reply:
Yes, I've fixed this in 8f47c5b.